### PR TITLE
Add v2 docs for color transforms

### PR DESCRIPTION
**Pull request by NicolasHug**

### CODE REVIEW

**Review by vfdev-5 — APPROVED** [review]
LGTM, thanks @NicolasHug !

File: torchvision/transforms/v2/_color.py [hunk 1]
```diff
@@ -199,10 +224,10 @@ class RandomPhotometricDistort(Transform):
 
     def __init__(
         self,
+        brightness: Tuple[float, float] = (0.875, 1.125),
```
Comment (pmeier) [inline torchvision/transforms/v2/_color.py:227]: This is just to align the signature with the logic below, right?

**Review by pmeier — APPROVED** [review]
One minor nit, otherwise LGTM. Thanks Nicolas!

File: torchvision/transforms/v2/_color.py (hunk 1 above)
Comment (NicolasHug) [reply torchvision/transforms/v2/_color.py:227]: yes sorry I forgot to comment - this is to align with the signature of ColorJitter

**Pull request merged**